src/components/scs/ownScs/OwnScsContainer.js

```diff
@@ -16,21 +20,26 @@ const styles = theme => ({
   padding: 3 * theme.spacing.unit
 });
 
-const OwnScsContainer = ({ classes, intl }) => {
+const OwnScsContainer = ({ classes }) => {
   const [ownScs, setOwnScs] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
 
   const error = useErrorContext();
+  const userId = useUserinfoContext().context.value.userinfo.userId;
 
   useEffect(() => {
     getOwnScs(setOwnScs, setIsLoading, error);
   }, []);
 
-  const listofOwnScs = ownScs.map((sc, index) => (
-    <Grid item key={index} className={classes.padding}>
-      <ScCard sc={sc} active={true} />
-    </Grid>
-  ));
+  const listofOwnScs = ownScs.map((sc, index) => {
+    const statuses = sc.statusSet;
+    const isOwner = userId === sc.employeeId;
```

Comment:
there is already such method in useUserinfoContext you are using:
change line 28: const user = useUserinfoContext();
and then instead of isOwner just: user.isOwnerInSc(sc)

Reply:
This sc is a different kind of object, so user.isOwnerInSc(sc) and other methods won't work. In ownScs and processingScs we don't have sc.employee.id or sc.reviewer1.id as this is required. Therefore the output is always undefined.

Reply:
If we want to use these methods, I would suggest to update responses from backend.

Reply:
You re right, then forget it, it s okay right now :D